gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Sun Jun 16 15:57:45 CDT 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6-bin.zip
```

Comment:
please revert this change, we should not be upgrading the projects gradle version for this task